nuxeo-core/nuxeo-core-mongodb/src/test/resources/OSGI-INF/mongodb-enable-unique-indices-contrib.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.test.mongodb.repository.indices">
+  <extension target="org.nuxeo.ecm.core.storage.mongodb.MongoDBRepositoryService" point="repository">
+    <repository name="test" label="MongoDB Repository">
```

Comment:
You can remove the `label` here as you are merging with an existing contribution.